constructor_io/__init__.py

```diff
@@ -1,3 +1,3 @@
-from constructor_io import *
-__version__ = '0.0.5'
+from .constructor_io import *
```

Comment:
Why did you make this import relative?

Reply:
I wanted to avoid uncertainty. Imagine such a situation:
- There is `constrictor_io` folder in current PYTHONPATH.
- The same time `constrictor_io` is installed.

This file is located in `/lib/python*.*/site-packages/constrictor_io/__init__.py`. 

I'm not sure which library will be imported in this case. Is it possible that when there is
```
from constructor_io inport ConstructorIO
```
interpreter will go to local packages, find there __init__.py and import `constrictor_io` folder from current PYTHONPATH?

May be I'm overthinking